subprojects/ide-native/src/main/java/org/gradle/ide/xcode/internal/XcodeScheme.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2017 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.ide.xcode.internal;
+
+import org.gradle.api.Named;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * @see <a href="https://developer.apple.com/library/content/featuredarticles/XcodeConcepts/Concept-Targets.html">XCode Target Concept</a>
+ */
+public class XcodeScheme implements Named {
```

Comment:
Not sure I would bother with this at this stage, and would generate a hard-coded scheme.